flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/MatchRecognizeITCase.scala

```diff
@@ -364,6 +364,69 @@ class MatchRecognizeITCase(backend: StateBackendMode) extends StreamingWithState
     assertEquals(expected.sorted, sink.getAppendResults.sorted)
   }
 
+  @Test
+  def testWindowedGroupingAppliedToMatchRecognizeOnLtzRowtime(): Unit = {
```

Comment:
How can we get deterministic results for every time zone for TIMESTAMP_LTZ rowtime?